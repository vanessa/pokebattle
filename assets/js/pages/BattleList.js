import React from 'react';

const BattleList = () => <h2>Your battles</h2>;

export default BattleList;
