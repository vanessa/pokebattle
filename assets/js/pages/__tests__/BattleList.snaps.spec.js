import React from 'react';
import renderer from 'react-test-renderer';
import { NotConnectedBattleList } from '../BattleList';

jest.mock('../../utils/urls');

describe('BattleList', () => {
  let Component;
  let tree;

  it('renders', () => {
    Component = renderer.create((
      <NotConnectedBattleList
        loadBattleList={jest.fn()}
      />
    ));

    tree = Component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
