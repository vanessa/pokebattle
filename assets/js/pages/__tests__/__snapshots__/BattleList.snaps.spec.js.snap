// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BattleList renders 1`] = `
<div
  className="centered-spinner"
  dangerouslySetInnerHTML={
    Object {
      "__html": "<svg xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 32 32\\">
  <path opacity=\\".25\\" d=\\"M16 0 A16 16 0 0 0 16 32 A16 16 0 0 0 16 0 M16 4 A12 12 0 0 1 16 28 A12 12 0 0 1 16 4\\"/>
  <path d=\\"M16 0 A16 16 0 0 1 32 16 L28 16 A12 12 0 0 0 16 4z\\">
    <animateTransform attributeName=\\"transform\\" type=\\"rotate\\" from=\\"0 16 16\\" to=\\"360 16 16\\" dur=\\"0.8s\\" repeatCount=\\"indefinite\\" />
  </path>
</svg>
",
    }
  }
  style={
    Object {
      "fill": "#ccc",
      "height": 70,
      "width": 70,
    }
  }
/>
`;
